lib/gradebook.rb

```diff
@@ -0,0 +1,29 @@
+class Gradebook
+  attr_reader :name, :courses
+  def initialize(name)
+    @name = name
+    @courses = []
+  end
+
+  def add_course(course)
+    @courses << course
+  end
+
+  def lis_all_students
```

Comment:
This doesn't match the specification, be super careful as to making your method names match what the spec demands.